### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by ShaniGam** [junyanz/pytorch-CycleGAN-and-pix2pix#158]
Hi,
Out of the losses: D_A, G_A, Cyc_A (and same to B), what does supposed to go down and what does suppose to go up (or stay the same) after training? I'm guessing that G and Cyc supposed to decrease substantially.

**Comment by junyanz** [comment]
Usually, cyc losses should go down gradually. D and G losses sometimes just go up and down. It is fine as long as they do not explode.

**Comment by ShaniGam** [comment]
Thank you!
